src/components/ItemList.svelte

```diff
@@ -85,7 +88,7 @@
         background: rgba($color-link-hover, 0.1);
       }
       .description {
-        @include text-body-sm;
+        @include text-body-xs;
```

Comment:
This is too small imo, to the point where it's slightly hard to read. I think we should keep the old size, and make a few compromises in other places. 
```suggestion
        @include text-body-sm;
```